game/resource/addon_english.txt

```diff
@@ -280,164 +280,164 @@
+        "Script_5_Roshan_1" "OK, let’s talk about the horrible monster you just saw. His name is Roshan and he lives int he pit. Let's go inside."
+        "Script_5_Roshan_2" "Roshan is a big badass that will beat anyone who attacks him to death in his little nerd cave. He's probably pretty angry because he gets no Wifi signal in there. To defeat him, most teams will potentially need a bunch of heroes in the area to attack him and take him down. But, we've been talking to you for so long, we thought that you would like to fight him alone for fun."
+        "Script_5_Roshan_3" "Here are some hyper late game items for Dragon Knight as well as a few level ups. Have fun!"
         "Script_5_Roshan_4" "Oh dear that is a lot of levels. So many that now, you have been gifted talents! Remember when we said that each hero comes with innate skills, but items can change the hero dynamically to be something different? Well talents do the same! At levels 10, 15, 20 and 25 you are given a choice for an upgrade, and those choices let you make the hero fit your playstyle. Select some talents you think you would like!"
         "Script_5_Roshan_5" "OK, that's all the talents, the items, and the confidence we can give you. DEFEAT ROSHAN!"
-        "Script_5_Roshan_6" "OH ITS YOUR FRIENDS, THEY HAVE COME TO DO BATTLE WITH YOU!"
-        "Script_5_Roshan_7" "Well done! Before you lies the aegis of immortality. Whoever picks it up is blessed with an extra life. This is very useful for doing things like winning team fights and doing that final push! Go ahead and take it"
+        "Script_5_Roshan_6" "OH IT'S YOUR FRIENDS, THEY HAVE COME TO DO BATTLE WITH YOU!"
+        "Script_5_Roshan_7" "Well done! Before you lies the Aegis of Immortality. Whoever picks it up is blessed with an extra life. This is very useful for doing things like winning team fights and doing that final push! Go ahead and take it"
         "Script_5_Roshan_8" "Wonderful wonderful. If only there was a way we could test this out. Feel free to leave the pit."
         "Script_5_5v5_1" "Oh dear, the time has finally come. It is time to use all we have taught you. Time for a TEAM FIGHT! LEARNING IS OVER! KILL THEM ALL!"
-        "Script_5_5v5_2" "Oh wow...he's dead. Wow i was not expecting that. Are we bad teachers?"
-        "Script_5_5v5_3" "No no no, that guy...that guy was a lost cause my brother. I mean did you see him just kinda flailing around? Poor bastard. That's the 5th player this week."
-        "Script_5_5v5_4" "Oh well. Maybe try candy crush or something! Bye!"
+        "Script_5_5v5_2" "Oh wow... he's dead. Wow I was not expecting that. Are we bad teachers?"
+        "Script_5_5v5_3" "No no no, that guy... that guy was a lost cause my brother. I mean did you see him just kinda flailing around? Poor bastard. That's the 5th player this week."
+        "Script_5_5v5_4" "Oh well. Maybe try Candy Crush or something! Bye!"
```

Comment:
These "fail" lines are not implemented AFAIK, and probably shouldn't anyway. Can we delete them from the sheet?